src/components/Nav.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => {
+  const navLinks = [
+    {
+      id: 1,
+      name: 'Home',
+      path: '/',
+    },
+    {
+      id: 2,
+      name: 'Calculator',
+      path: '/calculator',
+    },
+    {
+      id: 3,
+      name: 'Quote',
+      path: '/quote',
+    },
+  ];
+
+  return (
+    <nav className="NavItems flex">
+      <h3>Math Magicians</h3>
+      <ul>
+        {navLinks.map(({ id, name, path }) => (
+          <li key={id}>
+            <NavLink to={path}>{name}</NavLink>
+          </li>
+        ))}
+      </ul>
+    </nav>
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirement, according to which your design should match the wireframe styling, no doubt you did a great job but you just need to fix the navigation styling, your navigation has space at the top, which looks unprofessional and also breaching the `wireframe` requirement.

![Screenshot from 2023-09-22 05-21-25](https://github.com/Kingno3l/Math-Magician/assets/66476329/8770df8f-22d2-4545-ae11-bbc64778ab09)

![Screenshot from 2023-09-22 05-19-47](https://github.com/Kingno3l/Math-Magician/assets/66476329/34f1dc3f-ab45-4ed6-8cbb-96d27941147e)

Reply:
<img width="1440" alt="Screenshot 2023-09-22 at 14 58 55" src="https://github.com/Kingno3l/Math-Magician/assets/66368728/922d93f4-53c1-4769-ae7b-b454c3f58da7">
Thank you for the review, I edited the design. 👍